com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -137,18 +138,30 @@ public ConfigStyle(string label, string error, string button = resolve, MessageT
+                label: "Screen Space Shadows (Default Camera Frame Setting)",
+                error: "Screen Space Shadows are disabled in the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > Screen Space Shadows", messageType: MessageType.Info);
             public static readonly ConfigStyle dxrReflections = new ConfigStyle(
-                label: "Reflections",
-                error: "Screen Space Reflections are disabled in the current HDRP asset. You will not be able to toggle ray traced reflections though your volume components. You can enable the feature in the HDRP asset under Lighting -> Reflections -> Screen Space Reflections", messageType: MessageType.Info);
+                label: "Reflection (Asset)",
+                error: "Screen Space Reflection is disabled in the current HDRP asset. You will not be able to toggle ray traced reflections though your volume components. You can enable the feature in the HDRP asset under Lighting > Reflections > Screen Space Reflections", messageType: MessageType.Warning);
+            public static readonly ConfigStyle dxrReflectionsFS = new ConfigStyle(
+                label: "Reflection (Default Camera Frame Setting)",
+                error: "Screen Space Reflection is disabled in the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > Screen Space Reflections", messageType: MessageType.Info);
             public static readonly ConfigStyle dxrTransparentReflections = new ConfigStyle(
-                label: "Transparent Reflections",
-                error: "Transparent Screen Space Reflections are disabled in the current HDRP asset. You will not be able to toggle ray traced reflections on transparent objects though your volume components. You can enable the feature in the HDRP asset under Lighting -> Reflections -> Transparent Screen Space Reflections", messageType: MessageType.Info);
+                label: "Transparent Reflection (Asset)",
+                error: "Transparent Screen Space Reflection is disabled in the current HDRP asset. You will not be able to toggle ray traced reflections on transparent objects though your volume components. You can enable the feature in the HDRP asset under Lighting > Reflections > Transparent Screen Space Reflections", messageType: MessageType.Warning);
+            public static readonly ConfigStyle dxrTransparentReflectionsFS = new ConfigStyle(
+                label: "Transparent Reflection (Default Camera Frame Setting)",
+                error: "Transparent Screen Space Reflection is disabled in the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > On Transparent", messageType: MessageType.Info);
```

Comment:
Transparent Screen Space Reflection is disabled in the default Camera Frame Settings. This means Cameras that use these Frame Settings do not render ray-traced reflections for transparent GameObjects. To enable this feature, go to Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting and enable On Transparent.